_docs/concepts/policy-and-control/mixer.md

```diff
@@ -88,7 +88,7 @@ operator is responsible for:
 
 - Configuring a set of *handlers* for Mixer-generated data. Handlers are
   configured adapters (adapters being binary plugins as described
-  [below](#adapters)). Providing a `statsd` adapter with the IP address for a
+  [here](#adapters)). Providing a `statsd` adapter with the IP address for a
```

Comment:
1. Used "here" instead of "above" to avoid future new occurrences of similar semantics trickies
2. Tests for this would be an overkill, right?